config/default.json

```diff
@@ -14,7 +14,15 @@
       "dynamicConfigContinueOnError": false,
       "dynamicConfigExclude": ["ssr", "storeViews", "entities", "localForage", "shipping", "boost", "query"],
       "dynamicConfigInclude": [],
-      "elasticCacheQuota": 3096
+      "elasticCacheQuota": 4096,
+      "customErrors": {
+        "http404": {
+          "notAllowedExtensions": [".png", ".gif", ".jpg", ".jpeg", ".woff", ".eot", ".woff2", ".ttf", ".svg", ".css", ".js", ".json", ".ico", ".tiff", ".tif", ".raw"]
+        }
+      },
+      "ssrDisabledFor": {
+        "extensions": [".png", ".gif", ".jpg", ".jpeg", ".woff", ".eot", ".woff2", ".ttf", ".svg", ".css", ".js", ".json", ".ico", ".tiff", ".tif", ".raw"]
```

Comment:
it was suggested by Piotr to have another one filter to prevent loading static content at all for some specific extensions.  onter one filter is related to verify if 404 has to be rendered.
i was thinking about this already and we probably can combine them but i'm not sure as it could be different cases.

Reply:
Done.

Reply:
I'd really would stay with single setting of this and possibly extent it that way, when somebody would have a real need to do that kind of stuff.
@pkarw could you add your opinion why to have two separate settings? I'm afraid its harder for user to get what is what that way.

Reply:
Correct. `ssrDisabledFor` is more than enough. Le'ts keep it simple @yuriboyko

Reply:
Done